server/src/main/java/com/cloud/resourcelimit/ResourceLimitManagerImpl.java

```diff
@@ -1152,11 +1459,365 @@ public void changeResourceCount(long accountId, ResourceType type, Boolean displ
     }
 
     @Override
-    public ResourceReservation getReservation(final Account account, final Boolean displayResource, final Resource.ResourceType type, final Long delta) throws ResourceAllocationException {
-        if (! Boolean.FALSE.equals(displayResource)) {
-            return new CheckedReservation(account, type, delta, reservationDao, resourceLimitService);
+    public List<String> getResourceLimitHostTags() {
+        if (StringUtils.isEmpty(ResourceLimitService.ResourceLimitHostTags.value())) {
+            return new ArrayList<>();
+        }
+        return Arrays.asList(ResourceLimitService.ResourceLimitHostTags.value().split(","));
```

Comment:
can we use trim() here something like 

           List<String> ResourceLimitHostTags = Stream.of(ResourceLimitService.ResourceLimitHostTags.value().split(","))
                    .map(item -> (item).trim())
                    .collect(Collectors.toList());